test/common.js

```diff
@@ -88,6 +88,11 @@ exports.validate = (t, report, options) => {
       t.match(nodeReportSection,
               new RegExp('NodeReport version: ' + nodereportMetadata.version),
               'NodeReport section contains expected NodeReport version');
+      const sysInfoSection = getSection(reportContents, 'System Information');
+      // Find a line which ends with /api.node or \api.node (Unix or Windows paths.)
+      // to see if the library for node report was loaded.
+      t.match(sysInfoSection, /  .*[\/|\\]api\.node/,
```

Comment:
It should be`(\/|\\)`, we only want to match one kind of path separator. / does need escaping then as it's the delimiter for the regex.